bootstrap/config.go

```diff
@@ -18,3 +20,24 @@ func LoadConfig() {
 		log.Fatal("Failed to load config: ", err)
 	}
 }
+
+func LoadENV() {
```

Comment:
```suggestion
func LoadEnv() {
```

As the file `.env` is downcase 🤔